docs/content/tutorials/kic/deploy-with-kic.md

```diff
@@ -141,20 +79,20 @@ If you would like to enable egress traffic, refer to the [Enable Egress](#enable
 ### Install with Manifests
 
 Before installing NGINX Plus Ingress Controller, you must install NGINX Service Mesh with an [mTLS mode]({{< ref "/guides/secure-traffic-mtls.md" >}}) of `permissive`, or `strict`.
-NGINX Plus Ingress Controller will try to fetch certs from the SPIRE agent on startup. If it cannot reach the SPIRE agent, startup will fail, and NGINX Plus Ingress Controller will go into CrashLoopBackoff state. The state will resolve once NGINX Plus Ingress Controller connects to the SPIRE agent.
+NGINX Plus Ingress Controller will try to fetch certs from the SPIRE agent on startup. If it cannot reach the SPIRE agent, startup will fail, and NGINX Plus Ingress Controller will go into CrashLoopBackoff state. The state will resolve once NGINX Ingress Controller connects to the SPIRE agent.
 For instructions on how to install NGINX Service Mesh, see the [Installation]({{< ref "/get-started/install.md" >}}) guide.
 
 {{< note >}}
-Before continuing, check the NGINX Plus Ingress Controller [supported versions](#supported-versions) section and make sure you are working off the correct release tag for all NGINX Plus Ingress Controller instructions.
+Before continuing, check the NGINX Ingress Controller [supported versions](#supported-versions) section and make sure you are working off the correct release tag for all NGINX Ingress Controller instructions.
 {{< /note >}}
 
-1. Build or Pull the NGINX Plus Ingress Controller image:
+1. Build or Pull the NGINX OSS/Plus Ingress Controller image:
```

Comment:
Since these instructions are very tailored for the Plus version, it may just make more sense to have a different section for OSS. Does a user even need to build OSS? Plus is special since a user has to build it themselves with their cert/key for N+

Reply:
I broke every occurrence of this into separate sections.